### DISCUSSION TITLE
I need some help with a hacker that is here online daily

### DISCUSSION THREAD

**Discussion by Mike2337long** [community/community#130650] [New to GitHub]
Anybody Who can help me?? 

Info trough PM or asked me for mail.

HELP NEEDED! 

Sorry but thank you, really Need some help, cant point a finger here only,  only thing i know is that hé is getting all his info from here to ruin dozen of people lives..

**Comment by ghostinhershell** [comment]
Hi @Mike2337long, 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!

**Comment by ghost** [comment]
This dumb ass ass fuck my butt up I couldn't watch TV for two months

**Comment by ghost** [comment]
I can't wait when I finish going through it. I gotta go through the deactivate this son of a bitch ass page this son of a bitch ass ass.App Ppl you need to say fuck get her and Relly get something little deactivate. This page ASAP is not good.
